service/amf_server.go

```diff
@@ -24,17 +24,14 @@ type Server struct {
 }
 
 func (s *Server) HandleMessage(srv sdcoreAmfServer.NgapService_HandleMessageServer) error {
-	var Amf2RanMsgChan chan *sdcoreAmfServer.AmfMessage
-	Amf2RanMsgChan = make(chan *sdcoreAmfServer.AmfMessage, 100)
+	var Amf2RanMsgChan chan *sdcoreAmfServer.AmfMessage = make(chan *sdcoreAmfServer.AmfMessage, 100)
```

Comment:
What about the following for simplicity?
```suggestion
	Amf2RanMsgChan := make(chan *sdcoreAmfServer.AmfMessage, 100)
```